styles/core.scss

```
@@ -137,3 +137,139 @@ SPDX-License-Identifier: Apache-2.0 */
     }
   }
 }
+
+.core-page-title {
+  background: var(--background-default);
+  position: sticky;
+  top: 0;
+  padding-top: 0.5rem;
+  margin-top: 4rem;
+  margin-bottom: 0.25rem;
+  padding-bottom: 0;
```

Is this zero by default anyway?
```suggestion
  padding-bottom: 0;
```